apps/web/app/courses/[courseCode]/page.tsx

```diff
@@ -51,6 +60,14 @@ export default function CourseReview({ params }: { params: { courseCode: string
 				setReviewsData(reviews);
 			})
 			.catch(console.error);
+
+		useEffect(() => {
+			return () => {
+				localStorage.removeItem("reviewFormAcademicYear");
+				localStorage.removeItem("reviewFormDescription");
+				localStorage.removeItem("reviewFormRating");
+			};
+		}, [params.courseCode]);
```

Comment:
Pretty sure these are no longer used, not to mention the useEffect here. Can just delete this chunk I think